doc/src/fix_bond_react.txt

```diff
@@ -309,7 +313,16 @@ edge are unaffected by this setting.
 
 A few other considerations:
 
-It may be beneficial to ensure reacting atoms are at a certain
+Many reactions result in one or more atoms that are considered
+unwanted by-products. Therefore, bond/react provides the option to
+delete a user-specified set of atoms. These pre-reaction atoms are
+identified in the map file. A deleted atom must still be included in
+the post-reaction molecule template, in which it cannot be bonded to
+an undeleted atom. In addition to deleting unwanted reaction
```

Comment:
`undeleted` also shows up in the spellcheck part of the documentation build test. however, my impression is, that `undeleted` (i.e. previously deleted and now revived) may not be the correct term, but `not deleted` or `remaining` or something similar. can you please clarify? if undelete is the correct term, please also add to the list of false positives.

Reply:
yes, you're right, thanks